src/test/java/uk/nhs/prm/e2etests/tests/ContinuityE2E.java

```diff
@@ -252,13 +239,13 @@ public void shouldDeleteEhrOfPatientOnTheirReRegistration() throws Exception {
 
 
 
-        await().atMost(20, TimeUnit.SECONDS).untilAsserted(() -> { // check it's a 404?
-            assertThrows(HttpClientErrorException.class, () -> {
-                ehrRepoClient.getEhrResponse(patientNhsNumber);
-            });
+
+        assertThrows(HttpClientErrorException.class, () -> {
+            ehrRepoClient.getEhrResponse(patientNhsNumber);
```

Comment:
I believe we should revert to the original. This test was always failing even after the change.